### PR TITLE
Add min width to portion cells and examples

### PR DESCRIPTION
**Pull request by renanpvaz**
#### :thinking: What?
Fixing the issue where the portion columns were overflowing
Adding an example to cover this use case

#### :pushpin: Jira Issue
[WMS-473](https://paacklogistics.atlassian.net/browse/WMS-473)

### CODE REVIEW

File: cypress/integration/visual-regression.spec.js [hunk 1]
```diff
@@ -64,9 +64,26 @@ describe('Percy Snapshots', () => {
...
+
+      it('Creates the Stateless Tables page snapshot', () => {
+        cy.visit('/#Complex%20components/Tables/Stateless')
+        cy.percySnapshot('Tables Stateless')
+      })
+
+      it('Creates the Portion Column Tables page snapshot', () => {
+        cy.visit('/#Complex%20components/Tables/Portion%20Column')
+        cy.percySnapshot('Tables Portion Column')
+      })
```
Comment (renanpvaz) [inline cypress/integration/visual-regression.spec.js:86]: should we continue to add every case like this or does it make sense to make something that visits all pages and stories creating snapshots?

Comment (jouderianjr) [reply cypress/integration/visual-regression.spec.js:86]: I liked it this way, seems good to me. WDYT?

File: showcase/src/Tables/Stories.elm [hunk 2]
```diff
@@ -114,7 +115,17 @@ statelessDemoTable renderConfig =
 selectableTableStory renderConfig =
     storyWithModel
         ( "Selectable"
-        , \{ tablesStories } -> withIcons renderConfig (demoTable renderConfig tablesStories.selecTableState Stories.ForSelectable)
+        , \{ tablesStories } -> withIcons renderConfig (demoTable renderConfig Book.tablePixelColumns tablesStories.selecTableState Stories.ForSelectable)
```
Comment (jouderianjr) [inline showcase/src/Tables/Stories.elm:118]: I would break this line, it is too long I think

File: showcase/src/Tables/Stories.elm [hunk 3]
```diff
@@ -114,7 +115,17 @@ statelessDemoTable renderConfig =
...
+        , { defaultWithMenu
+            | note = "See [docs](https://package.elm-lang.org/packages/PaackEng/paack-ui/latest/UI-Tables-Stateful) for the exact code of this example."
+          }
+        )
+
+
+portionColumnsTable renderConfig =
+    storyWithModel
+        ( "Portion Columns"
+        , \{ tablesStories } -> withIcons renderConfig (demoTable renderConfig Book.tablePortionColumns tablesStories.selecTableState Stories.ForSelectable)
```
Comment (jouderianjr) [inline showcase/src/Tables/Stories.elm:128]: Same here

**Review by jouderianjr — APPROVED** [review]

File: cypress/integration/visual-regression.spec.js (hunk 1 above)
Comment (PedroHLC) [reply cypress/integration/visual-regression.spec.js:86]: :smile: I don't have an opinion right now, but I think this discussion should be moved to its own PR as this fix is quite urgent.

**Pull request merged**